Sources/SwiftKuery/ForeignKey.swift

```diff
@@ -90,3 +79,31 @@ struct ForeignKey: Hashable, Buildable {
         return append
     }
 }
+
+extension ForeignKey: Hashable {
+
+    #if swift(>=4.2)
+    func hash(into hasher: inout Hasher) {
+        let baseHash = "foreignKey".hashValue
+        hasher.combine(baseHash)
+        for key in keyNames {
+            hasher.combine(baseHash ^ key.hashValue)
+        }
+        for ref in refNames {
+            hasher.combine(baseHash ^ ref.hashValue)
+        }
+    }
```

Comment:
@kilnerm  Though I don't see any problem with this implementation, is it not enough to have something like:
```
    func hash(into hasher: inout Hasher) {
        hasher.combine("foreignKey")
        for key in keyNames {
            hasher.combine(key)
        }
        for ref in refNames {
            hasher.combine(ref)
        }
    }
```

Reply:
I thought the same but decided to keep things as similar to the previous implementation as possible.